internal/commands/update_stemcell.go

```diff
@@ -12,14 +12,16 @@ import (
 
 	"github.com/pivotal-cf/kiln/internal/commands/flags"
 	"github.com/pivotal-cf/kiln/internal/component"
+	"github.com/pivotal-cf/kiln/pkg/cargo"
 )
 
 type UpdateStemcell struct {
 	Options struct {
 		flags.Standard
 
-		Version     string `short:"v"  long:"version"            required:"true"    description:"desired version of stemcell"`
-		ReleasesDir string `short:"rd" long:"releases-directory" default:"releases" description:"path to a directory to download releases into"`
+		Version        string `short:"v"   long:"version"               required:"true"    description:"desired version of stemcell"`
+		ReleasesDir    string `short:"rd"  long:"releases-directory"    default:"releases" description:"path to a directory to download releases into"`
+		UpdateReleases bool   `long:"update-releases"   description:"finds latest matching releases for new stemcell version"`
```

Comment:
Could you add the "short" for this as well? 

`ur` would match the scheme of the other flags